plugins/modules/zos_copy.py

```diff
@@ -234,6 +223,88 @@
         unit name has been specified.
     type: str
     required: false
+  type:
+    description:
+      - If the destination does not exist and is not a USS file, this determines
+        the type of dataset created.
+      - The data set type to be used when creating a data set. (e.g C(pdse))
+      - C(MEMBER) expects to be used with an existing partitioned data set.
+      - Choices are case-insensitive.
+    required: false
+    type: str
+    choices:
+      - KSDS
+      - ESDS
+      - RRDS
+      - LDS
+      - SEQ
+      - PDS
+      - PDSE
+      - LIBRARY
```

Comment:
I can remove library (don't recognize it).  I don't believe there is any mount-related logic in here.  The only bad thing there is if we pre-delete (which we do) a mounted entry, I'm not sure how that would end up.  I'll remove them, to be safe.

Reply:
DSNTYPE=LIBRARY is used to define PDSE, as opposed to DSNTYPE=PDS to define PDS so it's ok to remove LIBRARY from this list

Reply:
@richp405 you are correct we don't have any mount logic, I think i was just going off to far left and thinking back to the discussions to support HFS/zFS and how a simple thing becomes complicated because now you potentially have to deal with mounts for those types, thus was getting to that fact I was not familiar with `library` type till Andy commented, and such a simple thing as Library could drag in many corner case as HFS/zFS might. 

Thanks marking resolved.